parsers/src/main/scala/uk/gov/ons/addressIndex/parsers/Tokens.scala

```diff
@@ -1,29 +1,67 @@
 package uk.gov.ons.addressIndex.parsers
 
-import uk.gov.ons.addressIndex.crfscala.CrfScala.{CrfToken, CrfTokenable, CrfTokens, Input}
+import com.typesafe.config.ConfigFactory
+import uk.gov.ons.addressIndex.crfscala.CrfScala.CrfTokenable
 
-//todo scaladoc this thing
+import scala.io.Source
+
+/**
+  * Tokenizer for the input
+  */
 object Tokens extends CrfTokenable {
```

Comment:
Should Tokens be a Class with instantiation params for the underlying file resources?
This would remove the usage of config in this class and move the concern of the config in the server from the lib impl